File: csharp/armstrong-numbers/ArmstrongNumbers.cs
using System;
using System.Linq;

public static class ArmstrongNumbers
{
    public static bool IsArmstrongNumber(int number)
    {
        if (number.ToString().Select(n => Math.Pow(Char.GetNumericValue(n), number.ToString().Length)).Sum() == number)
            return true;
        return false;
    }
}

File: javascript/armstrong-numbers/armstrong-numbers.js
//
// This is only a SKELETON file for the 'Armstrong Numbers' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const isArmstrongNumber = (number) => {
  let digits = [...number.toString()].map(nr => parseInt(nr));
  let digitsPoweredSum = digits.reduce((prevVal, currVal) => prevVal + Math.pow(currVal, digits.length), 0);
  return number === digitsPoweredSum;
};
